### DISCUSSION TITLE
plugins: preview-tui: sox as alternative for mediainfo

### DISCUSSION THREAD

**Discussion by vlappa** [jarun/nnn#1841] [Ideas]
For audio file I think sox gives a nicer preview then mediainfo does for audio files.

It seems that wav files are handles by the print_bin_info() function

```# Binary file: show file info inside the pager
print_bin_info() {
    printf -- "-------- \033[1;31mBinary file\033[0m --------\n"
    if exists sox; then
        soxi "$1"
    elif exists mediainfo; then
        mediainfo "$1"
    else
        file -b "$1"
    fi
}

**Comment by N-R-K** [comment]
That `print_bin_info()` function isn't just for audio files. It's for any binary files in general.
